README.md

```diff
@@ -58,7 +58,7 @@ You can also exclude specific data from the restore. For example, if
 you want to restore only MySQL data, and leave the state of MongoDB
 and Caddy untouched, run:
 
-    tutor k8s restore --exclude=caddy --exclude=mongodb
+    tutor local restore --exclude=caddy --exclude=mongodb
```

Comment:
Whoops, sorry about that. Thanks for the fix. :)

Reply:
No worries! If only the reviewer of that PR was more attentive ... :smile: